app/controllers/publish/courses/course_information_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Publish
+  module Courses
+    class CourseInformationController < PublishController
+      def edit
+        authorize(provider)
+
+        @course_information_form = CourseInformationForm.new(course.model.latest_published_enrichment)
+      end
+
+      def update
+        authorize(provider)
+
+        @course_information_form = CourseInformationForm.new(course.model.latest_published_enrichment, params: course_information_params)
+
+        if @course_information_form.save!
+          flash[:success] = I18n.t("success.published")
```

Comment:
![image](https://user-images.githubusercontent.com/470137/154272795-a32dd7d3-a5a6-4e20-a737-8ae11c2b13b4.png)

vs

![image](https://user-images.githubusercontent.com/470137/154272464-cf3786cc-7b5f-4798-a566-a0e0cfbb1c8a.png)


The original behaviour seems to have been changed.

Reply:
fixed https://github.com/DFE-Digital/teacher-training-api/pull/2519/commits/7ea4dd2bcafa21fa24f218ee821321a84d94e028